cmd/root.go

```diff
@@ -78,7 +78,16 @@ var RootCmd = &cobra.Command{
 			return err
 		}
 
-		return initApp(cmd, defaultClient, defaultViper)
+		err := initApp(cmd, defaultClient, defaultViper)
+		// When no config file has been read and initApp returns an error, tell
+		// the user how to initialize the application.
+		if err != nil && defaultViper.ConfigFileUsed() == "" {
+			return multierror.NewPrefixed(
+				`missing ecctl config file, please use "ecctl init" to initialize it`, err,
```

Comment:
@marclop just a minor fix - "... please use "ecctl init" command, to initialize ecctl"